@ornikar/eslint-config-react/tests/Typography.js

```diff
@@ -20,7 +20,9 @@ Typography.div.propTypes = {
   children: PropTypes.node,
 };
 
-const InvalidFunctionComponent = () => null;
+function InvalidFunctionComponent() {
+  return null;
+}
```

Comment:
```suggestion
// eslint-disable-next-line react/function-component-definition
const InvalidFunctionComponent = () => null;
```

c'est le test ! faut le garder :)

tu peux en rajouter un autre pour unnamedComponents ?

Reply:
Ok j'update ça